lib/cli/changeset.js

```diff
@@ -71,20 +72,27 @@ SubCommand.prototype.process = function(proc, callback) {
 		}
 	});
 
-	var stdin = proc.stdin
-		.pipe(Diff.stream({
-			ignoreWhitespace: !!self.opts['--ignore-whitespace']
-		}))
-		.pipe(MetadataContainer.diffStream())
+	var diff = {}; // first get the stdin stream with all git diff changes
+	proc.stdin.pipe(miss.concat(function(result) {
+		diff = Diff.parse(result, { ignoreWhitespace: !!self.opts['--ignore-whitespace'] });
+	}));
 
-	mergeStream(stdin, metadataContainer.getStream())
```

Comment:
@amtrack, why `mergeStream` exists? I removed this merge and it seems to be working...
If it is not needed, then we can add a commit removing `metadataContainer.getStream()` function.